redaxo/src/core/pages/packages.list.php

```diff
@@ -161,6 +152,10 @@
     </script>
 ';
 
+if (rex_api_function::hasMessage()) {
+    echo rex_api_function::getMessage();
+}
```

Comment:
das if ist überflüssig, wenn man die message einfach nur ausgibt ohne eigenen wrapper.

```suggestion
echo rex_api_function::getMessage();
```